contracts/Gauge.sol

```diff
@@ -1,98 +1,115 @@
     struct Checkpoint {
-        uint timestamp;
-        uint balanceOf;
+        uint256 timestamp;
+        uint256 balanceOf;
     }
 
     /// @notice A checkpoint for marking reward rate
     struct RewardPerTokenCheckpoint {
-        uint timestamp;
-        uint rewardPerToken;
+        uint256 timestamp;
+        uint256 rewardPerToken;
     }
 
     /// @notice A checkpoint for marking supply
     struct SupplyCheckpoint {
-        uint timestamp;
-        uint supply;
+        uint256 timestamp;
+        uint256 supply;
     }
 
     /// @notice A record of balance checkpoints for each account, by index
-    mapping (address => mapping (uint => Checkpoint)) public checkpoints;
+    mapping(address => mapping(uint256 => Checkpoint)) public checkpoints;
     /// @notice The number of checkpoints for each account
-    mapping (address => uint) public numCheckpoints;
+    mapping(address => uint256) public numCheckpoints;
     /// @notice A record of balance checkpoints for each token, by index
-    mapping (uint => SupplyCheckpoint) public supplyCheckpoints;
+    mapping(uint256 => SupplyCheckpoint) public supplyCheckpoints;
     /// @notice The number of checkpoints
-    uint public supplyNumCheckpoints;
+    uint256 public supplyNumCheckpoints;
     /// @notice A record of balance checkpoints for each token, by index
-    mapping (address => mapping (uint => RewardPerTokenCheckpoint)) public rewardPerTokenCheckpoints;
+    mapping(address => mapping(uint256 => RewardPerTokenCheckpoint))
+        public rewardPerTokenCheckpoints;
     /// @notice The number of checkpoints for each token
-    mapping (address => uint) public rewardPerTokenNumCheckpoints;
-
-    uint public fees0;
-    uint public fees1;
-
-    event Deposit(address indexed from, uint tokenId, uint amount);
-    event Withdraw(address indexed from, uint tokenId, uint amount);
-    event NotifyReward(address indexed from, address indexed reward, uint amount);
-    event ClaimFees(address indexed from, uint claimed0, uint claimed1);
-    event ClaimRewards(address indexed from, address indexed reward, uint amount);
-
-    constructor(address _stake, address _internal_bribe, address _external_bribe, address  __ve, address _voter, bool _forPair, address[] memory _allowedRewardTokens) {
+    mapping(address => uint256) public rewardPerTokenNumCheckpoints;
+
+    uint256 public fees0;
+    uint256 public fees1;
+
+    event Deposit(address indexed from, uint256 tokenId, uint256 amount);
+    event Withdraw(address indexed from, uint256 tokenId, uint256 amount);
+    event NotifyReward(
+        address indexed from,
+        address indexed reward,
+        uint256 amount
+    );
+    event ClaimFees(address indexed from, uint256 claimed0, uint256 claimed1);
+    event ClaimRewards(
+        address indexed from,
+        address indexed reward,
+        uint256 amount
+    );
+
+    constructor(
+        address _stake,
+        address _internal_bribe,
```

Comment:
[Question] Are we going to keep internal bribe as parameter in constructor here?